Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["src/firecracker", "src/jailer", "src/seccompiler", "src/rebase-snap", "src/cpu-template-helper"]
+members = ["src/cpu-template-helper", "src/firecracker", "src/jailer", "src/rebase-snap", "src/seccompiler"]
```

Comment:
I think this diff is also due to cargo-sort.

Reply:
Yeah, no idea how this one snuck in :( Should be fixed now though!